_includes/releases/v23.1/v23.1.0.md

```diff
@@ -231,6 +231,10 @@ Asyncpg is commonly used with ORM libraries such as SQLAlchemy to provide a simp
 
 <h5>Change data capture (Changefeeds)</h5>
 
+{{site.data.alerts.callout_info}}
```

Comment:
Fine for now, but we have an include for this.

Reply:
It's edited text from the include